bundles/org.openhab.ui/web/src/components/widgets/system/oh-swiper.vue

```diff
@@ -24,12 +29,14 @@ import { OhSwiperDefinition } from '@/assets/definitions/widgets/system'
 
 import OhSwiperSlide from './oh-swiper-slide.vue'
 import OhPlaceholderWidget from '../layout/oh-placeholder-widget.vue'
+import GenericWidgetComponent from '../generic-widget-component.vue'
 
 export default {
   mixins: [mixin],
   components: {
     OhSwiperSlide,
-    OhPlaceholderWidget
+    OhPlaceholderWidget,
+    GenericWidgetComponent
```

Comment:
You don't need to reference this GenericWidgetComponent because it's one of the few that are registered globally:

https://github.com/openhab/openhab-webui/blob/dd6e756382891e568957819b255ae4e1684accdc/bundles/org.openhab.ui/web/src/js/app.js#L82

Usually the reason is because they can be direct or indirect children of themselves.

Reply:
oh, so that's why it worked when during development I found the import line to be present (I must have inserted it somewhen and forgot about it), but the GenericWidgetComponent to not be registered as component 😄.
I just removed it.